collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormMetadataTest.kt

```diff
@@ -25,12 +25,10 @@ class FormMetadataTest {
     fun startGeopoint_withCachedFormDefinition_doesNotCauseError() {
         rule.startAtMainMenu()
             .copyForm("start-geopoint.xml")
-
```

Comment:
They just decided we shouldn't use blank lines like that https://github.com/pinterest/ktlint/issues/1248
I could disable the rule but it would also mean disabling checking blank lines in all the other cases (that I like).

Reply:
I feel pretty strongly about keeping those blank lines. In a lot of case it might not make sense, but I do think it helps readability in our UI tests when related groups are broken up by a blank line like that. Can we put ignore the rule and revert related changes?

Reply:
No because the rule is `no-consecutive-blank-lines` so as I said if you disable it you disable checking all the other cases where having blank lines is obviously a bad thing.

Reply:
Sorry what I mean is that I'd be happy to just disable that rule. I feel like the blank lines are useful enough in some places to make up for other places they might appear and not look quite right.

Reply:
Are you sure? I'm going to repeat: if we do that you can add multiple blank lines in many different ways in the codebase and it won't be detected. I didn't want to do that because in all the other cases detecting blank lines worked well.

Reply:
Ah, you mean it's not just for chains? Sorry you're right I'm misunderstanding and that it's not worth disabling.

Reply:
You know what? now as I downgraded ktlint because of that problem with imports (see above) I can bring those lines back too. So maybe let's do that for now and complain about that in their repo (and ask if we can disable this particular check somehow if they are unwavering). What do you think?

Reply:
Yeah, agreed! It does seem a little broken as a line followed by a comment shouldn't count as "consecutive" blank lines. I can post a reply (seeing as I seem to care so much 😆).

Reply:
Ok I reverted that commit.